src/typedKnex.ts

```diff
@@ -1290,7 +1292,6 @@ class TypedQueryBuilder<ModelType, SelectableModel, Row = {}> implements ITypedQ
             let columnAlias;
             let currentClass;
             let currentColumnPart;
-            // console.log('keys: ', keys);
```

Comment:
I'm assuming this can be removed, unless you had a reason to keep it in there?

Reply:
I'll miss it ...